pypeman/channels.py

```diff
@@ -84,6 +84,7 @@ def __init__(self, name=None, parent_channel=None, loop=None, message_store_fact
         if name:
             self.name = name
         else:
+            # TODO: shall we force set name param and raise an exc ?
```

Comment:
I suggest to check since which release this is considered obsolete and change the message into something like.

... is obsolete since vxxx and will be deprecated in vyyyy 


yyy shall be the next version

Reply:
There's a warning the line after that says it's already deprecated

Reply:
you can change the warning to:
```
warnings.warn("Channels without names are deprecated since version XXX and will be removed in version YYY", DeprecationWarning)
```

Reply:
Changed